netbox/extras/choices.py

```diff
@@ -117,10 +117,12 @@ class BookmarkOrderingChoices(ChoiceSet):
 
     ORDERING_NEWEST = '-created'
     ORDERING_OLDEST = 'created'
+    ORDERING_NAME_AZ = 'name'
 
     CHOICES = (
         (ORDERING_NEWEST, _('Newest')),
         (ORDERING_OLDEST, _('Oldest')),
+        (ORDERING_NAME_AZ, _('Name (A-Z)')),
```

Comment:
I'm not sure how useful it would be, but it feels like we should add an option for reverse alphabetical ordering as well for completeness.